magento/magento2
# Simplify setup wizard: remove angular, combine all data on one page, etc

### Description (*)

Simplify setup wizard code:
- remove angular
- combine landing+license on the same page
- remove other unused files

This could be one of the steps to remove laminas MVC and other dependencies used only for setup scripts and which are not necessary after removing the main Setup Wizard logic

### Related Pull Requests


### Fixed Issues (if relevant)


### Manual testing scenarios (*)

Open setup wizard `/setup/` and check that both landing and license pages are the same as before

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#32550: Simplify setup wizard: remove angular, combine all data on one page, etc
